pkg/mgrconfig/load.go

```diff
@@ -237,6 +237,10 @@ func (cfg *Config) completeServices() error {
 func (cfg *Config) initTimeouts() {
 	slowdown := 1
 	switch {
+	case cfg.Type == "qemu" && runtime.GOARCH == "arm64" && cfg.SysTarget.Arch == "arm64":
```

Comment:
This should check VMArch instead of Arch.

Reply:
Removed this condition